packages/lib/date-fns/index.ts

```diff
@@ -226,3 +226,18 @@ export const isInDST = (date: Dayjs) => {
 
   return timeZoneWithDST(timeZone) && date.utcOffset() === getUTCOffsetInDST(timeZone);
 };
+
+/**
+ * Get UTC offset of given time zone
+ * @param timeZone Time Zone Name (Ex. America/Mazatlan)
+ * @returns
+ */
+export function getUTCOffsetByTimezone(timeZone: string) {
+  if (!timeZone) return null;
+
+  if (timeZoneWithDST(timeZone)) {
+    return getUTCOffsetInDST(timeZone);
```

Comment:
Wouldn't this now always return DST offset of this timezone, even when we are not in DST like now in December?

Reply:
Wow, you reminded me so well. I was overthinking it. This is not necessary.